HACKING.md

```diff
@@ -131,7 +100,7 @@ installed (since it will not get updates as it was installed dangerously), you
 can either use `snap revert snapd`, or you can refresh directly with 
 `snap refresh snapd --stable --amend`.
 
-Note: It is also sometimes useful to use snapcraft to build the snapd snap for
+> It is also sometimes useful to use snapcraft to build the snapd snap for
 other architectures using the `remote-build` feature, however there is currently a
 bug in snapcraft around using the `4.x` track and using `remote-build`, where the
 Launchpad job created for the remote-build will attempt to use the `latest` track instead
```

Comment:
So, just to make sure I understand you correctly, @anonymouse64. 

You suggestion is to remove the text starting from `However, this fix does...` (until the end of the paragraph). 
since this explicit removal of these files is not longer needed. Do I understand correctly?

If yes, then I also think that we need to remove the following command line too, as after the fix it's done automatically:
```
rm -r tests/main/validate-container-failures/
```
Am I right?

Reply:
Yes that is all correct